Kudu.Core/Infrastructure/SecurityUtility.cs

```diff
@@ -35,8 +35,15 @@ private static string GenerateSecretString()
 
         public static string DecryptSecretString(string content)
         {
-            var protector = DataProtectionProvider.CreateAzureDataProtector().CreateProtector(DefaultProtectorPurpose);
-            return protector.Unprotect(content);
+            try
+            {
+                var protector = DataProtectionProvider.CreateAzureDataProtector().CreateProtector(DefaultProtectorPurpose);
+                return protector.Unprotect(content);
+            }
+            catch (CryptographicException)
+            {
+                throw new FormatException($"unable to decrypt {content}, the key is either invalid or malformed");
```

Comment:
Shouldn't the original CryptographicException be passed in as the inner exception?

            catch (CryptographicException e) {
                throw new FormatException($"unable to decrypt {content}, the key is either invalid or malformed", e);